packages/api-clients/open-api/delegationApi.yml

```diff
@@ -37,6 +37,58 @@ tags:
       description: Find out more
       url: http://swagger.io
 paths:
+  /consumer/eservices:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+      - in: query
+        name: eserviceName
+        schema:
+          type: string
+      - in: query
+        name: delegateId
+        required: true
+        schema:
+          type: string
+          format: uuid
```

Comment:
This is used instead of the organizationId? Is it ok or should we use the organizationId also in the delegation process?

Reply:
> This is used instead of the organizationId? Is it ok or should we use the organizationId also in the delegation process?

No, you're right, it should be the `organizationId`, I misread the task

Reply:
> This is used instead of the organizationId? Is it ok or should we use the organizationId also in the delegation process?

IMHO this route should be generic, so we can retrieve the eservices of a `delegateId` being passed as parameter.

For this specific task the specific router on the BFF has the job to set the `delegateId` to the `organizationId` of the requester

Reply:
I would also by default restrict the call to its known usages instead of making it more generic without knowing if it will be used like that - also, by removing the delegated param here and using the requester, we can apply a check that requester is the delegate, which I think is something that should be done at the service level. Now that I think about it, the same should be done for the get consumer delegators in my opinion (implemented in a previous PR: #1295)

From now on, given the discussions we had, this is the general approach I would follow when implementing new API calls:
- always apply visibility permissions using assertions on requester tenant
- don't make them more generic than it's strictly needed: in this way it will always be clear that if a call has a parameter/filter it's because it is used, otherwise we could end up in cases like the agreements/purposes listing that are super generic and we don't know anymore where/how they are used

For calls that are already implemented, we will add the visibility checks in the future as discussed